ui/packages/catalog-ui-search/src/main/webapp/component/filter/filter.less

```diff
@@ -44,9 +44,6 @@
       max-width: 300px;
     }
   }
-}
-
-@{customElementNamespace}filter.is-sortable {
```

Comment:
Removed this because this line was a tautology: https://github.com/codice/ddf/pull/5099/files#diff-ac37e8e62b0c35f5a2a198bdd21c08a2L161

Reply:
it looks like an `isSortable` attribute is still being passed in https://github.com/codice/ddf/blob/master/ui/packages/catalog-ui-search/src/main/webapp/component/filter/filter.collection.view.js#L36 should we correct the logic instead of removing it?

Reply:
The tautology has been there since the query form has existed, so I'll just remove the prop entirely.